lib/ship_fosdick/downloader.rb

```diff
@@ -0,0 +1,21 @@
+module ShipFosdick
+  class Downloader
+    def initialize(bucket)
+      @bucket = bucket
+    end
+
+    def download
+      [].tap do |returned_files|
+        bucket.objects.each do |object|
+          next unless object.key.downcase.include?('shp')
```

Comment:
Good call @DanielWright, @braidn see this: https://github.com/DynamoMTL/fosdick_integration/commit/578b8fc0200dc11f5a47f6c45d56585c4ae041db